.github/REUSE.toml

```diff
@@ -0,0 +1,11 @@
+version = 1
+
+[[annotations]]
+path = ["*", "**/"]
+SPDX-FileCopyrightText = "Copyright © 2022 The OSRD Contributors"
+SPDX-License-Identifier = "LGPL-3.0-only"
```

Comment:
I'm pretty sure `technical_drawing/1.png` isn't licensed under the LGPL, and that we do not hold the copyright to it.